### PR TITLE
[Choreo] Enable Backend JWT per Api

### PR DESCRIPTION
**Pull request by jhivandebenoit**
### Purpose
Backend JWT can be turned on/off per api through the api artifacts. Default is on.

### Automation tests
 - Unit tests added: Yes/No
 - Integration tests added: Yes/No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [x] Assigned 'Type' label
- [x] Assigned the project
- [x] Validated respective github issues
- [x] Assigned milestone to the github issue(s)

### CODE REVIEW

File: adapter/internal/oasparser/model/mgw_swagger.go [hunk 1]
```diff
@@ -1221,6 +1222,9 @@ func (swagger *MgwSwagger) PopulateSwaggerFromAPIYaml(apiData APIYaml, apiType s
 	// context value in api.yaml is assigned as xWso2Basepath
 	swagger.xWso2Basepath = data.Context + "/" + swagger.version
 
+	// Enable/Disable Backend JWTs per api
+	swagger.DisableBackendJWT = data.DisableBackendJWT
```
Comment (pubudu538) [inline adapter/internal/oasparser/model/mgw_swagger.go:1226]: If this new property is not present we have to set the default value as true. Let's add test cases as well.

Comment (jhivandebenoit) [reply adapter/internal/oasparser/model/mgw_swagger.go:1226]: We want backend JWT to be disabled by default? Ack on adding test cases.

Comment (jhivandebenoit) [reply adapter/internal/oasparser/model/mgw_swagger.go:1226]: Added test case

**Review by VirajSalaka — APPROVED** [review]

**Review by pubudu538 — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[failed] DataplaneE1 cluster : dev-deployment-v2 : 20230308.21

**Comment by choreo-cicd** [comment]
[failed]  : dev-deployment-v2 : 20230308.21

**Comment by choreo-cicd** [comment]
[failed] Controlplane cluster : dev-deployment-v2 : 20230308.21

**Comment by choreo-cicd** [comment]
[succeeded]  : dev-deployment-v2 : 20230308.24

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : dev-deployment-v2 : 20230308.24

**Comment by choreo-cicd** [comment]
[succeeded] DataplaneE1 cluster : dev-deployment-v2 : 20230308.24

**Comment by choreo-cicd** [comment]
[failed]  : stage-deployment-v2 : 20230309.2

**Comment by choreo-cicd** [comment]
[succeeded]  : stage-deployment-v2 : 20230309.2

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : stage-deployment-v2 : 20230309.2

**Comment by choreo-cicd** [comment]
[failed] Controlplane cluster : prod-deployment-v2 : 20230310.1

**Comment by choreo-cicd** [comment]
[failed] Dataplane(CentralIN) cluster : prod-deployment-v2 : 20230310.1

**Comment by choreo-cicd** [comment]
[failed] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20230310.1

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20230310.1

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20230310.1

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(CentralIN) cluster : prod-deployment-v2 : 20230310.1

**Pull request merged**